CONTRIBUTING.md

```diff
@@ -121,6 +121,42 @@ cabal test ouroboros-consensus:test:consensus-test --test-show-details=direct
 Note the second one cannot be used when we want to provide CLI arguments to the
 test-suite.
 
+# Generating documentation and setting up hoogle
```

Comment:
We should create an issue for providing a `nix` alternative to these steps (and assign it to @amesgen :grimacing: )